### PR TITLE
lego1: match MxBitmap::Read and vtable

### PR DESCRIPTION
**Pull request by foxtacles**
Matched and cleaned up `MxBitmap::Read` (and implemented `MxBitmap::vtable28`), also added the stubs for the other functions. Their signatures aren't necessarily 100% correct yet, just based on a cursory glance.

### CODE REVIEW

**Comment by ItzSwirlz** [comment]
thank you! ill check in on this later today

File: LEGO1/mxbitmap.cpp [hunk 1]
```diff
@@ -14,15 +14,75 @@ MxBitmap::MxBitmap()
 // OFFSET: LEGO1 0x100bca10
 MxBitmap::~MxBitmap()
 {
-  if (this->m_info != NULL) {
+  if (this->m_info != NULL)
```
Comment (ItzSwirlz) [inline LEGO1/mxbitmap.cpp:17]: yeah we need a clang-format thing, i'll open a pr for it

**Comment by ItzSwirlz** [comment]
this looks good! I'm going to open some PR's to your branch if you're okay with that, Sorry I forgot the vtables..

**Comment by foxtacles** [comment]
> this looks good! I'm going to open some PR's to your branch if you're okay with that, Sorry I forgot the vtables..

Sure thing, and no problem! Maybe wait until this PR is merged into `master` and then make your PRs against that. I think that would be easier

File: LEGO1/mxbitmap.cpp (hunk 1 above)
Comment (foxtacles) [reply LEGO1/mxbitmap.cpp:17]: Should probably discuss this with @itsmattkc first if we want that and if yes, how it should be configured

**Comment by itsmattkc** [comment]
Looks good to me. And yeah I think further PRs on this can be made to `master` here. Merging now.

**Pull request merged**